src/FS.Embedded/build/netstandard2.0/Microsoft.Extensions.FileProviders.Embedded.props

```diff
@@ -0,0 +1,6 @@
+<Project>
```

Comment:
Moved this a props file so the csproj can access the name for instance if it chooses to.